pkg/storkctl/applicationclone_test.go

```diff
@@ -0,0 +1,227 @@
+	cmdArgs := []string{"create", "clones"}
+
+	expected := "error: exactly one name needs to be provided for applicationclone name"
+	testCommon(t, cmdArgs, nil, expected, true)
+}
+
+func TestCreateApplicationClones(t *testing.T) {
+	defer resetTest()
+	createApplicationCloneAndVerify(t, "createclone", "default", "src", "dest", "", "")
+}
+
+func TestCreateDuplicateApplicationClones(t *testing.T) {
+	defer resetTest()
+	createApplicationCloneAndVerify(t, "createclone", "default", "src", "dest", "", "")
+	cmdArgs := []string{"create", "clones", "createclone", "--sourceNamespace", "src", "--destinationNamespace", "dest"}
+
+	expected := "Error from server (AlreadyExists): applicationclones.stork.libopenstorage.org \"createclone\" already exists"
+	testCommon(t, cmdArgs, nil, expected, true)
+}
+
+func TestDeleteApplicationClonesNoApplicationCloneName(t *testing.T) {
+	cmdArgs := []string{"delete", "clones"}
+
+	var cloneList storkv1.ApplicationCloneList
+	expected := "error: at least one argument needs to be provided for applicationclone name"
+	testCommon(t, cmdArgs, &cloneList, expected, true)
+}
+
+func TestDeleteApplicationClones(t *testing.T) {
+	defer resetTest()
+	createApplicationCloneAndVerify(t, "deleteclone", "default", "src", "dest", "", "")
+
+	cmdArgs := []string{"delete", "clones", "deleteclone"}
+	expected := "ApplicationClone deleteclone deleted successfully\n"
+	testCommon(t, cmdArgs, nil, expected, false)
+
+	cmdArgs = []string{"delete", "clones", "deleteclone"}
+	expected = "Error from server (NotFound): applicationclones.stork.libopenstorage.org \"deleteclone\" not found"
+	testCommon(t, cmdArgs, nil, expected, true)
+
+	createApplicationCloneAndVerify(t, "deleteclone1", "default", "src", "dest", "", "")
+	createApplicationCloneAndVerify(t, "deleteclone2", "default", "src", "dest", "", "")
+
+	cmdArgs = []string{"delete", "clones", "deleteclone1", "deleteclone2"}
+	expected = "ApplicationClone deleteclone1 deleted successfully\n"
+	expected += "ApplicationClone deleteclone2 deleted successfully\n"
+	testCommon(t, cmdArgs, nil, expected, false)
+
+	createApplicationCloneAndVerify(t, "deleteclone1", "default", "src", "dest", "", "")
+	createApplicationCloneAndVerify(t, "deleteclone2", "default", "src", "dest", "", "")
+}
+
+func TestCreateApplicationCloneWaitSuccess(t *testing.T) {
+	cloneStatusRetryInterval = 10 * time.Second
+	defer resetTest()
+
+	srcNamespace := "dummy-src-namespace"
+	dstNamespace := "dummy-dst-namespace"
+	name := "dummy-name"
+	cmdArgs := []string{"create", "clones", "-n", srcNamespace, "--sourceNamespace", srcNamespace,
+		"--destinationNamespace", dstNamespace, name, "-w"}
+
+	expected := "ApplicationClone dummy-name started successfully\n" +
+		"STAGE\t\tSTATUS              \n" +
+		"\t\t                    \n" +
+		"Volumes\t\tSuccessful          \n" +
+		"ApplicationClone dummy-name completed successfully\n"
+	go setApplicationCloneStatus(name, srcNamespace, false, t)
```

Comment:
I'm planning to customise retryTimeout and retryInterval for unit test purpose 
eg. `storkctl create migrations -w 2,10 (2 seconds retry, 10 Min wait)

Reply:
Unit tests can change private variables so we can just reduce the variables for the poll times. I already reduced it for the migration UT